formats/mainstream_browse_page/publisher/details.json

```diff
@@ -3,6 +3,9 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "child_ordering": {
+      "type": "string"
```

Comment:
Instead of a string, this would be better as an `enum`.  See https://github.com/alphagov/govuk-content-schemas/blob/master/formats/redirect/publisher/schema.json#L21-L23 for example.